standalone/src/models/store/store/store-draft/presets/empty.spec.ts

```diff
@@ -1,8 +1,39 @@
-import { TStoreDraft } from '../../types';
-import empty from './empty';
+import { TStoreDraftGraphql, TStoreDraftRest } from '../../types';
+import { emptyDraftRest, emptyDraftGraphql, empty } from './empty';
 
 it('should set all specified fields to undefined', () => {
-  const emptyStoreDraft = empty().build<TStoreDraft>();
+  const emptyStoreDraft = empty().build<TStoreDraftRest | TStoreDraftGraphql>();
```

Comment:
By default, `build()` generates a rest model:
```suggestion
  const emptyStoreDraft = empty().build<TStoreDraftRest>();
```